site/build/handlebars-helpers/hyperlink.js

```diff
@@ -0,0 +1,19 @@
+
+function register(handlebars) {
+    handlebars.registerHelper('hyperlink', function(link, context) {
+        var pages = context.data.root.pages;
+        var matches = pages.filter(function(pageMetadata) {
+            return pageMetadata.page.destination.endsWith('/' + link);
+        });
+        if (matches.length === 1) {
+            var title = context.hash.title;
```

Comment:
Worth checking `context.hash.title` is not `undefined` and has length `> 0` - otherwise throw an error?

Not sure if it's worth trimming any whitespace? `title.trim()`